articles/logs/streams/splunk.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Splunk Event Log Streams
+description: Learn about Splunk Event Log Streams and how they let you export your events in near real-time to Splunk.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Splunk Event Log Streams
+
+Splunk is the Data-to-Everything Platform turns data into action, tackling the toughest IT, IoT, security and data challenges.
+
+You can create monitoring, alerting, and analysis dashboards for Auth0 tenants.
+
+
+## Send events from Auth0 to Splunk
+
+To send Auth0 events to Splunk, you will need:
+
+1. Splunk instance domain name
+2. Splunk event collector token
+3. Port
+
+
+#### Retrieving the Domain, Token, and Port
+
+  1. Navigate to your Splunk instance. You will have received this information via email upon Splunk signup. Copy the domain part of the URL, this is your Splunk **Domain**.
```

Comment:
If the customer has installed and setup their own Splunk Enterprise instance, they might not get the signup email.

Reply:
@xer0x if that case, where can they get the domain name? do you know

Reply:
@xer0x and @sharmoo let me know what you think of this change

Reply:
aside from @xer0x 's comment above (https://github.com/auth0/docs/pull/9144/files/0d0de591ef867f49e86758aa6c9d170fb978b427#r438456958), looks good.

Reply:
regarding splunk enterprise or managed accounts, the customer may want to reference: https://docs.splunk.com/Documentation/Splunk/8.0.4/Data/UsetheHTTPEventCollector#Send_data_to_HTTP_Event_Collector_on_Splunk_Enterprise but the docs are buggy so I don't feel comfortable expanding upon them beyond a link (our instructions for even the extensions are far more clear)

Reply:
@sharmoo Thanks! I addressed both of @xer0x comments. and I agree that explaining the Enterprise flow is not necessary. Would you mind approving?